charts/portal/templates/deployment-backend-administration.yaml

```diff
@@ -38,6 +38,10 @@ spec:
         app: {{ include "portal.fullname" . }}-{{ .Values.backend.administration.name }}
         {{- include "portal.selectorLabels" . | nindent 8 }}
     spec:
+      {{- with .Values.backend.administration.image.pullSecrets }}
+      pullSecrets:
```

Comment:
the value in the deployment still needs to be names `imagePullSecrets`, otherwise K8s won't use it; just the path for templating coming from the values.yaml should be named `pullSecrets`